doc/design/web.md

```diff
@@ -0,0 +1,153 @@
+    cpu_per_ps,
+)
+```
+
+After this, there will be a job description and performance monitoring pages able to view at "Job Dashboard"
+
+## Job Dashboard
+
+### Job history and currently running jobs
+
+A web page containing a table to list jobs satisfying user's filters. The user can only list jobs that were submitted by themselves.
+
+| jobname | start time | age      | success | fails | actions |
+| ------- | ---------- | -------- | ------- | ----- | ------- |
+| test1   | 2017-01-01 | 17m      |    0    |   0   | stop/log/perf |
+
+Users can filter the list using:
+
+- status:  running/stopped/failed
+- time:    job start time
+- jobname: search by jobname
+
+Viewing job logs:
+
+Click the "log" button on the right of the job will pop up a console frame at the bottom of the page showing the tail of the job log, here shows the first pod's log. On the left side of pop up console frame, there should be a vertical list containing the list of pods, then click one of the pod, the console will show it's log.
+
+### Performance Monitoring
+
+A web page containing graphs monitoring job's resource usages according to time change:
+
+- CPU usage
+- GPU usage
+- memory usage
+- network bandwidth
+- disk I/O
+
+### Quota Monitoring
+
+A web page displaying total quota and quota currently in use.
+
+Also display total CPU time, GPU time in latest 1day, 1week, and 1month.
+
+## Datasets and Models
+
+Datasets and Models are quite the same, both like a simple file management and sharing service.
+
+- file listing and viewing page
+- Upload/Download page
+- file sharing page
+
+## Paddle Board
+
+A web page containing graphs showing the internal status when the job is training, metrics like:
+
+- cost(can be multiple costs)
+- evaluator output
+- user-defined metric
+
+User can caculate metrics and define the graph like:
+
+```python
+cost = my_train_network()
+evaluator = my_evaluator(output, label)
+def my_metric_graph(output, label):
+    metric = paddle.auc(output, label)
+my_metric = my_metric_graph(output, label)
+my_metric_value = output
+
+draw_board(cost, evaluator)
+draw_board(my_metric)
+draw_board(my_metric_value)
+```
+
+Calling `draw_board` will output graph files on the distributed storage, and then the web page can load the data and refresh the graph.
+
+## Serving
+
+After training or uploading pre-trained models, a user can start a serving instance to serve the model as an inference HTTP service:
+
+The Serving web page contains a table listing currently running serving instance and a "Launch" button to configure and start the serving program.
+
+Click the "Launch" button in this web page will pop up a modal dialogue to configure the job:
+
+1. model `tar.gz` files to the cloud.
+1. inference network configuration in `.proto` format or user can also define the network in Python in the web page.
+1. number of CPU/GPU resource in total to use for serving the model, the more resource there is, the more concurrent calls can be served.
```

Comment:
Should we change "number of CPU/GPU resource" to number of instances and CPU / Mem / GPU per instance. Otherwise it's hard for us to figure out how many instances to run (we don't know the model property and user's serving requirement).

Reply:
Agree with @helinwang , additional, we can caculate the total resources usage on the web site and display them on the web site.